### PR TITLE
Shikkic/GitHub lib indexer update

### PR DESCRIPTION
**Pull request by Shikkic**
Created RequestParams for Request Service. Now APIKeys have special properties surrounding it's for the indexer or not.

### CODE REVIEW

File: common/db/migrations/0006_github_api_add_for_indexer.up.cql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+------------------------------ GITHUB API KEY FOR INDEXER ----------------------------------
+
+ALTER TABLE gophr.github_api_key ADD for_indexer;
```
Comment (skeswa) [inline common/db/migrations/0006_github_api_add_for_indexer.up.cql]: Why don't you specify the data type of the column?

File: common/github/request_service.go [hunk 2]
```diff
@@ -9,10 +9,13 @@ import (
...
 // GitHubGophrPackageOrgName is the  Github organization name for all versioned packages
-// GitHubBaseAPIURL is the base Github API
 const (
 	GitHubGophrPackageOrgName = "gophr-packages"
-	GitHubBaseAPIURL          = "https://api.github.com"
+)
+
+// GitHubBaseAPIURL is the base Github API
+const (
+	GitHubBaseAPIURL = "https://api.github.com"
```
Comment (skeswa) [inline common/github/request_service.go]: Why is this in a separate const block?

File: common/db/migrations/0006_github_api_add_for_indexer.up.cql (hunk 1 above)
Comment (Shikkic) [inline common/db/migrations/0006_github_api_add_for_indexer.up.cql]: Whoooooops

File: common/github/request_service.go (hunk 2 above)
Comment (Shikkic) [inline common/github/request_service.go]: My linter was complaining about them needing export comments previously but when the comments are on top of each other they look really botched on GoDoc,  so I separated them.

**Pull request merged**